client-js/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "type": "module",
```

Comment:
Without changing the build to export an ESM compliant bundle, consumers of the package who have also declared type: module will error out as the only module type exported by the package is commonJS